src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -1356,6 +1357,45 @@ class misc_commandscript : public CommandScript
         return true;
     }
 
+    // kick player
+    static bool HandleKickDelayPlayerCommand(ChatHandler* handler, Optional<PlayerIdentifier> target, Optional<std::string_view> reason, uint32 min_time, uint32 max_time)
+    {
+        if (!target)
+        {
+            target = PlayerIdentifier::FromTargetOrSelf(handler);
```

Comment:
Why FromTargetOrSelf? are we meaning for the commands to kick or ban the user of the gm cmds?

Reply:
Good catch, this should just be FromTarget

Reply:
Should be okay now.